src/nl/help/guides/getting-started.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Aan de slag
+lang: nl-NL
+---
+
+# Aan de slag
+
+## Installatie
+**Paperback** is momenteel _niet_ op **TestFlight** en _vereist_ het gebruik van **AltStore** om op je iPhone of iPad te installeren. In de toekomst zal TestFlight worden gebruikt wanneer de hoofdontwikkelaar een [Apple Developer Account] heeft (https://developer.apple.com/programs/).
+
+::: tip Video tutorial
+Als je liever een video-tutorial gebruikt, heeft Artesians er een mooie gemaakt. Je vindt deze [hier] (https://www.youtube.com/watch?v=n1KRwsxNiWY).
```

Comment:
There is an issue with some links. You should remove the space characters for them to work:
 * `[Apple Developer Account](https://developer.apple.com/programs/)`
 * `[hier](https://www.youtube.com/watch?v=n1KRwsxNiWY)`
 * `[GitHub releases-pagina] (https://github.com/FaizanDurrani/Paperback/releases/latest)`
 * `[Afbeelding](https://imgur.com/a/onrwNC8)`

Reply:
Fixed

Reply:
Perfect.